Countly.js

```diff
@@ -127,7 +127,7 @@ Countly.hasBeenCalledOnStart = function () {
  *
```

Comment:
I guess this is draft and u will initialize the module later

Reply:
done